packages/cozy-pouch-link/src/CozyPouchLink.js

```diff
@@ -97,13 +105,38 @@ class PouchLink extends CozyLink {
     this.client = client
   }
 
+  async migrateAdapter({ fromAdapter, toAdapter, url, plugins }) {
```

Comment:
can you add the JSDoc please?

Reply:
Weird, I think I somehow missed a commit :thinking: 
Anyway fixed with ac29aac